ServerCore/Pages/Puzzles/Index.cshtml

```diff
@@ -91,8 +93,10 @@
                 <a asp-Page="./Details" asp-route-id="@item.ID">Details</a> |
                 <a asp-Page="./Delete" asp-route-id="@item.ID">Delete</a> |
                 <a asp-page="./Status" asp-route-id="@item.ID">Status</a> |
-                <a asp-Page="./Feedback" asp-route-id="@item.ID">Admin View Feedback</a> |
-                <a asp-Page="./SubmitFeedback" asp-route-id="@item.ID">User Submit Feedback</a> |
+                @{
+                    var feedbackLink = (role == "play") ? "./SubmitFeedback" : "./Feedback";
```

Comment:
Puzzles/Index is currently unused in the play role. We use Teams/Play instead. [Maybe these pages should be merged at some point, maybe not.]